boxoffice/templates/cash_receipt.html

```diff
@@ -212,7 +212,7 @@ <h1 class="header">Cash Receipt</h1>
       
       {% for line_item in line_items -%}
         <tr>
-          <td class="center" rowspan="2" valign="top">{{ line_item.line_item_seq }}</td>
+          <td class="center" rowspan="2" valign="top">{{ loop.index }}</td>
```

Comment:
We are using `loop.index` to indicate the row number of the table.
`line_item.line_item_seq` will not be in sequence incase we are cancelling bulk discounted tickets.